go/host/node/node.go

```diff
@@ -468,43 +475,33 @@ type blockAndParent struct {
 	p common.EncodedBlock
 }
 
-func (a *Node) processBlocks(blocks []common.EncodedBlock, interrupt *int32) error {
-	var result common.BlockSubmissionResponse
-	for _, block := range blocks {
-		// For the genesis block the parent is nil
-		if block == nil {
-			continue
-		}
+func (a *Node) processBlock(block common.EncodedBlock, latest bool, interrupt *int32) error {
```

Comment:
This method used to take a list of blocks (either a parent and child pair or a list of blocks from a fork).

It treated the last block in that list differently (error was more important and it bothered to publish the rollup). This change makes that explicit and hopefully easier to follow.